artemis/modules/lfi_detector.py

```diff
@@ -0,0 +1,119 @@
+import random
+import re
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+from urllib.parse import urlparse, urlunparse
+
+import more_itertools
+from karton.core import Task
+
+from artemis import load_risk_class
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.config import Config
+from artemis.crawling import get_links_and_resources_on_same_domain
+from artemis.http_requests import HTTPResponse
+from artemis.module_base import ArtemisBase
+from artemis.modules.data.lfi_detector_data import LFI_PAYLOADS
+from artemis.modules.data.static_extensions import STATIC_EXTENSIONS
+from artemis.sql_injection_data import URL_PARAMS
```

Comment:
if we use this in two modules, it can be extracted to e.g. artemis/modules/data/